README.md

```diff
@@ -45,11 +46,12 @@ go build grofer.go
 ---
 
 Shell Completions
```

Comment:
I've been thinking about this for a while now, can we shift the `shell completions` section to the bottom of the README? considering that its more of a convenience thing than a core functionality.

Reply:
Makes sense, I'll do that